src/XCMEnums.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.13;
+
+library XCMEnums {
```

Comment:
This library is a bit unnecessary. Can define the enums at the toplevel like so:

```solidity
// SPDX-License-Identifier: Apache-2.0
pragma solidity ^0.8.19;

enum XcmV3Junction {
    Parachain,
    AccountId32,
    AccountIndex64,
    AccountKey20,
    PalletInstance,
    GeneralIndex,
    GeneralKey,
    OnlyChild,
    Plurality,
    GlobalConsensus
}

```